spec/team_controller_spec.rb

```diff
@@ -51,24 +55,50 @@ def app
     end
   end
 
-  describe 'POST teams/' do
-    let(:team_object) { { id: team_one.id, type: 'team', name: request_body.name, league: team_one.league } }
+  describe 'POST /teams' do
+    let(:team_one) { create(:team, name: 'SV Hafen Rostock', league: 'Landesliga') }
+    let(:team_object) { { type: 'team', name: team_one.name, league: team_one.league } }
```

Comment:
You are inconsistent with the GET tests. Either provide the `team_object` as a hash in all tests, or as a JSON string in all tests.